lib/zold/commands/remote.rb

```diff
@@ -63,7 +63,9 @@ def run(args = [])
     #{Rainbow('remote elect').green}
       Pick a random remote node as a target for a bonus awarding
     #{Rainbow('remote trim').green}
-      Remote the least reliable nodes
+      Remove the least reliable nodes
+    #{Rainbow('remote select --max-nodes n').green}
```

Comment:
@botanicus no need to say it here. Just do `remote select [options]` as we do in all other commands